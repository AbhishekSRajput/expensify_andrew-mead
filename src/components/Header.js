import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
	<header>
		<h1>Expensify Home</h1>
		<NavLink to="/" activeClassName="is-active" exact>
			Dashboard
		</NavLink>
		<NavLink to="/create" activeClassName="is-active">
			Create_expense
		</NavLink>
		<NavLink to="/help" activeClassName="is-active">
			help
		</NavLink>
	</header>
);

export default Header;
