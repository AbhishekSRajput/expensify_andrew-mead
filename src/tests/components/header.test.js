import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../components/Header';

test('should render header correctly', () => {
	const renderer = new ReactShallowRenderer();
	renderer.render(<Header />);
	expect(renderer.getRenderOutput()).toMatchSnapshot();
	//lets you see the component you rendered and it's elements
	//console.log(renderer.getRenderOutput());
});
