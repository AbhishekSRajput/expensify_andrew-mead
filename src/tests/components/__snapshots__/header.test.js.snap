// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    Expensify Home
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create_expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    help
  </NavLink>
</header>
`;
